Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:bookworm
```

Comment:
why do we need the docker?

Reply:
We don't need Docker, but I needed Docker to run tests, because this project won't install on Arch Linux. I just put the Docker files in here because they don't hurt anything, and I figured somebody else may benefit from having a quick way to run tests.

Reply:
```suggestion
FROM ubuntu:22.04
```

Reply:
Added.